### DISCUSSION TITLE
Can't find a route for 'post: login' after customizing login and register routes

### DISCUSSION THREAD

**Discussion by anshu8858** [codeigniter4/shield#320] [Q&A]
Can't find a route for 'post: login'

Post method is not working in **Login**

I want to use CI Shield Post:login route but it does not display in route list.
Have to make custom login route for Post?
What I do? I want to customize default login view.

```php
$routes->get('login', 'AuthCtrlr::loginView');
$routes->get('register', 'AuthCtrlr::registerView');
service('auth')->routes($routes, ['except' => ['login', 'register']]);
// service('auth')->routes($routes);
```

**Comment by kenjis** [comment]
> Have to make custom login route for Post?

Yes. You need to define every route. If you miss one route, the route does not work.

You can check routes with `php spark routes`.
https://codeigniter4.github.io/CodeIgniter4/incoming/routing.html#confirming-routes

See also #296

**Reply by anshu8858** [reply]
okay
thanks for help.
